sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsClient.java

```diff
@@ -636,4 +636,54 @@ public void updateComponent(String digitalTwinId, String componentPath, List<Obj
     public DigitalTwinsResponse<Void> updateComponentWithResponse(String digitalTwinId, String componentPath, List<Object> componentUpdateOperations, UpdateComponentRequestOptions options, Context context) {
         return digitalTwinsAsyncClient.updateComponentWithResponse(digitalTwinId, componentPath, componentUpdateOperations, options, context).block();
     }
+
+    /**
+     * Query digital twins.
+     *
+     * @param query The query string, in SQL-like syntax.
+     * @return A {@link PagedIterable} of application/json query result items.
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public PagedIterable<String> query(String query) {
+        return new PagedIterable<>(digitalTwinsAsyncClient.query(query));
```

Comment:
nit: we've followed the pattern where the sync APIs call their next max overloads -> this sync API should call the sync API overload with Context.NONE.